spoolsense_installer/spoolman.py

```
@@ -3,51 +3,130 @@
+def print_failed_fields_summary(spoolman_url: str, failed: list) -> None:
+    """Print a prominent warning listing fields that could not be created.
+
+    Includes copy-paste curl commands and a pointer to the --setup-fields flag so
+    the failure is impossible to miss in busy install output.
+    """
+    if not failed:
+        return
+
+    field_meta = {(e, k): (ft, dn) for e, k, ft, dn in EXTRA_FIELDS}
+    print(f"\n{C.RED}{C.BOLD}{'═' * 42}")
+    print(f"  ⚠  Spoolman fields NOT created")
```

Ruff (0.15.20)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)